.appveyor.yml

```diff
@@ -3,28 +3,29 @@ version: 1.0.{build}
 clone_folder: c:\dev\lib-template-cmake
 
 os:
-    - Visual Studio 2013
-    - Visual Studio 2015
-    - Visual Studio 2017
+  - Visual Studio 2013
+  - Visual Studio 2015
+  - Visual Studio 2017
```

Comment:
Not directly related to the PR, but I just noticed that on VS we just test using the 32-bit platform, never compiling for 64-bit .

Reply:
🙈 
I'll add the tests right away, deprecating `Visual Studio 2013`.